lib/Dialect/ModArith/IR/ModArithOps.cpp

```diff
@@ -62,30 +49,14 @@ LogicalResult verifySameWidth(OpType op, ModArithType modArithType,
 LogicalResult ExtractOp::verify() {
   auto modArithType = getOperandModArithType(*this);
   auto integerType = getResultIntegerType(*this);
-  auto result = verifySameWidth(*this, modArithType, integerType);
-  if (result.failed()) return result;
-  return verifyModArithType(*this, modArithType);
-}
-
-LogicalResult ReduceOp::verify() {
-  return verifyModArithType(*this, getResultModArithType(*this));
+  return verifySameWidth(*this, modArithType, integerType);
 }
 
-LogicalResult AddOp::verify() {
-  return verifyModArithType(*this, getResultModArithType(*this));
-}
-
-LogicalResult SubOp::verify() {
-  return verifyModArithType(*this, getResultModArithType(*this));
-}
-
-LogicalResult MulOp::verify() {
-  return verifyModArithType(*this, getResultModArithType(*this));
-}
-
-LogicalResult MacOp::verify() {
-  return verifyModArithType(*this, getResultModArithType(*this));
-}
+LogicalResult ReduceOp::verify() { return success(); }
+LogicalResult AddOp::verify() { return success(); }
+LogicalResult SubOp::verify() { return success(); }
+LogicalResult MulOp::verify() { return success(); }
+LogicalResult MacOp::verify() { return success(); }
```

Comment:
The `let hasVerifier = 1;` line in `ModArithOps.td` for these ops could be deleted, and these `verify()` here could also be cleaned up.